dev/create-release/release-build.sh

```diff
@@ -835,11 +835,12 @@ if [[ "$1" == "docs" ]]; then
   PRODUCTION=1 RELEASE_VERSION="$SPARK_VERSION" bundle exec jekyll build
 
   # Generate llms.txt for LLM consumption
+  # Note this doc is currently versionless and need to be placed at the documentation root.
   echo "Generating llms.txt..."
   python "$SELF/generate-llms-txt.py" \
     --docs-path . \
-    --output "$SELF/_site/llms.txt" \
-    --version "$SPARK_VERSION"
+    --output "$SELF/llms.txt" \
```

Comment:
Does mean we should only do it when releasing the latest major/feature version? We should'n overwrite it with an old version when releasing 4.0.1 for example.